### ISSUE TITLE
TMC2209 V2 reporting error after update on Tevo Tornado

### ISSUE THREAD

**Issue by rajinder-yadav** [MarlinFirmware/Marlin#21464]
### Did you test the latest `bugfix-2.0.x` code?

Yes, and the problem still exists.

### Bug Description

After upload latest firmware build I am getting error report got my TMC2209 v2 driver

### Bug Timeline

New bug, Using latest firmware

### Expected behavior

TMC2209 driver should report no error

### Actual behavior

Send: M122
Recv: Testing X connection... Error: All LOW
Recv: Testing Y connection... Error: All LOW
Recv: Testing Z connection... Error: All LOW
Recv: Testing E connection... Error: All LOW

### Steps to Reproduce

Use gcode M122

### Version of Marlin Firmware

FIRMWARE_NAME:Marlin 2.0.7.2

### Printer model

Tevo Tornado V2

### Electronics

standard motherboard

### Add-ons

TMC2209

### Your Slicer

Cura

### Host Software

OctoPrint

Here is the following configuration: https://github.com/rajinder-yadav/Tevo-Tornado-Configuration

**Comment by ellensp** [comment]
This Issue Queue is for Marlin bug reports and development-related issues, and we prefer not to handle user-support questions here. (As noted on [this page](https://github.com/MarlinFirmware/Marlin/blob/2.0.x/.github/contributing.md#i-dont-want-to-read-this-whole-thing-i-just-have-a-question).) For best results getting help with configuration and troubleshooting, please use the following resources:

- MarlinFW.org [Marlin Documentation](http://marlinfw.org)
- RepRap.org [Marlin Forum](http://forums.reprap.org/list.php?415)
- [Tom's 3D Forums](https://discuss.toms3d.org/)
- Facebook Group ["Marlin Firmware"](https://www.facebook.com/groups/1049718498464482/)
- Facebook Group ["Marlin Firmware for 3D Printers"](https://www.facebook.com/groups/3Dtechtalk/)
- [Marlin Configuration](https://www.youtube.com/results?search_query=marlin+configuration) on YouTube
- Marlin Discord server. Join link: https://discord.gg/n5NJ59y

After seeking help from the community, if the consensus points to a bug in Marlin, then you should post a [bug report](https://github.com/MarlinFirmware/Marlin/issues/new/choose).

**Comment by ellensp** [comment]
This is always a wiring or configuration issue. not a marlin bug.

**Comment by rajinder-yadav** [comment]
Well this was not an problem with the previous Marlin firmware I had built and installed several weeks back. Using the same configurations settings and diffing files I've verified the configuration files changes and don't see anything off.

This showed up after using the latest firmware, so I see this as a legit issue.

**Comment by rhapsodyv** [comment]
If you switch now, for that old marlin version you said, everything works?

**Comment by rajinder-yadav** [comment]
@rhapsodyv so I think I know what I did, my board MKS Gen L V1.0 doesn't support UART mode, let me know if my fix make sense. What I ended up doing was comment out the drivers in Configuration.h

```cpp
// #define X_DRIVER_TYPE TMC2209
// #define Y_DRIVER_TYPE TMC2209
// #define Z_DRIVER_TYPE TMC2209
// #define E0_DRIVER_TYPE TMC2209
```

Am I correct to assume, the above line control UART mode?
Unfortunately I did a stupid mistake and deleted the old Marlin folder and I guess I never copied the correction (above) to my Config files I had in a separate folder. I am still not sure about the suggested change. I can't find anything that tells me what puts the firmware code into UART mode? other than the guess I suggested above.
